src/System.Management.Automation/engine/InitialSessionState.cs

```diff
@@ -5883,6 +5887,10 @@ internal static class PSSnapInHelpers
                         var aliasList = new List<SessionStateAliasEntry>();
                         foreach (var alias in aliasAttribute.AliasNames)
                         {
+                            // Alias declared by AliasAttribute is set with the option 'ScopedItemOptions.None',
+                            // because we believe a user of the cmdlet, instead of the author of it,
+                            // should be the one to decide the option
+                            // ('ScopedItemOptions.ReadOnly' and/or 'ScopedItemOptions.AllScopes') of the alias usage."
```

Comment:
This comment probably should be removed now. When making the change to add internal support to `AliasAttribute` to allow specifying options, we can have a more accurate comment here.